src/js/store/actions.ts

```diff
@@ -361,7 +363,7 @@ export const updateObject = ({
       url: baseUrl,
       dispatch,
       opts: {
-        method: 'PUT',
+        method: patch ? 'PATCH' : 'PUT',
```

Comment:
@wlonk This was working with a `PUT` and partial data, but I changed to `PATCH` since I think that's technically correct.